fahrplan/main.py

```diff
@@ -117,7 +124,7 @@ def assert_enough_arguments(args):
 
     url = '%s/connections' % API_URL
     try:
-        response = requests.get(url, params=args)
+        response = requests.get(url, params=args, proxies={'http' : proxy_host})
```

Comment:
Does this work properly if `proxy_host` is an empty stirng? Maybe this would be a bit nicer:

```
kwargs = {'params': args}
if proxy_host is not None:
    kwargs['http'] = proxy_host
response = requests.get(url, **kwargs)
```

And then initialize `proxy_host` with `None` instead of an empty string.